modules/va_notify/app/models/va_notify/veteran.rb

```diff
@@ -13,7 +13,7 @@ def initialize(in_progress_form)
     end
 
     def icn
-      @icn ||= in_progress_form&.user_account&.icn
+      @icn ||= in_progress_form.user_account&.icn
```

Comment:
Are there any cases when in_progress_form could be null? This could result in many errors.

Reply:
Not to my knowledge, given that other areas of the class call `in_progress_form.form_id`, `form_data`, & `user_uuid` there is not an expecatation of this I'd say.

From `modules/va_notify/app/sidekiq/va_notify/in_progress_1880_form_reminder.rb`
```ruby
      in_progress_form = InProgressForm.find(form_id)

      veteran = VANotify::Veteran.new(in_progress_form)

      return if veteran.first_name.blank?
      return if veteran.icn.blank?
```   
with a `null` in_progress_form attempting to call `first_name` would cause the same error on those other attributes.